### PR TITLE
Add 7plus and Fanatical as hard

### PR DESCRIPTION
**Pull request by SSSEAL-C**
Addition of a new site, 7plus.

Difficulty medium due to a form submission being needed, and having to find it on a support site.

### CODE REVIEW

**Review by tupaschoal — COMMENTED** [review]
But upon submitting the form, is the account closed immediately? Because if it's not, then it probably is a manual process and thus should be moved to hard.

Also, can you confirm whether closing equals deleting in this case, or just deactivating?

**Comment by SSSEAL-C** [comment]
I am currently checking, I cannot confirm deletion or deactivation as there is no real direct way to check.

**Comment by tupaschoal** [comment]
I see, but do you think deletion happen automatically via form submission or was it routed to a person?

File: _data/sites.json [hunk 1]
```diff
@@ -155,6 +155,15 @@
...
+    {
+        "name": "7plus",
+        "url": "https://support.7plus.com.au/hc/en-au/requests/new",
+        "difficulty": "hard",
+        "notes": "Create a new request, select the 'Reason For Contact' as 'Account Related' and then under 'Please choose the issue' select 'Close 7plus account'. Supply the appropriate email and name, then send the request.",
+        "domains": [
+            "7plus.com.au"
+         ]
+    },
     {
```
Comment (tupaschoal) [inline _data/sites.json:167]:
```suggestion
    },

    {
```

**Comment by SSSEAL-C** [comment]
I did also add Fanatical, apologies if I am doing this wrong I am unfamiliar with pull requests.

File: _data/sites.json [hunk 2]
```diff
@@ -4528,7 +4538,17 @@
             "fairphone.com"
         ]
     },
-
+    
+    {
+        "name": "Fanatical",
```
Comment (tupaschoal) [inline _data/sites.json:4543]:
```suggestion

    {
        "name": "Fanatical",
```

File: _data/sites.json [hunk 3]
```diff
@@ -4528,7 +4538,17 @@
...
+        "url": "https://support.fanatical.com/hc/en-us/requests/new",
+        "difficulty": "hard",
+        "notes": "Create a support ticket requesting for a closure of your account.",
+        "domains": [
+            "fanatical.com"
+        ]
+    },
+
```
Comment (tupaschoal) [inline _data/sites.json:4551]:
```suggestion
        ]
    },

```

**Review by tupaschoal — COMMENTED** [review]
Just needs to remove extra whitespaces

File: _data/sites.json [hunk 4]
```diff
@@ -155,6 +155,16 @@
...
     },
 
+    {
+        "name": "7plus",
+        "url": "https://support.7plus.com.au/hc/en-au/requests/new",
+        "difficulty": "hard",
+        "notes": "Create a new request, select the 'Reason For Contact' as 'Account Related' and then under 'Please choose the issue' select 'Close 7plus account'. Supply the appropriate email and name, then send the request.",
+        "domains": [
+            "7plus.com.au"
+         ]
```
Comment (tupaschoal) [inline _data/sites.json:165]:
```suggestion
        ]
```

**Review by tupaschoal — APPROVED** [review]

**Pull request merged**